src/domain/checklist/uploadToS3Checklist.ts

```diff
@@ -0,0 +1,52 @@
+import { pipe, tuple } from 'fp-ts/lib/function';
+import * as O from 'fp-ts/Option';
+import * as RA from 'fp-ts/ReadonlyArray';
+import { AnyRecord } from '../AnyRecord';
+import { PreLoadRecord } from '../PreLoadRepository';
+import { UploadToS3Record } from '../UploadToS3RecordRepository';
+import { Checklist } from './types';
+
+export const check0 = {
+  name: 'Exists a response with status code 200',
+  eval: RA.some((record: AnyRecord) => record.output.statusCode === 200 && record.type === 'UploadToS3Record'),
+};
```

Comment:
Following your suggestion we introduce an issue due to a lack of type checking. Once we enable the [`strict`](https://www.typescriptlang.org/tsconfig#strict) option this code should be changed because it will not compile anymore.

What do you think if we fix it after the task about the `strict` option?

Reply:
Totally agree